core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/runners/BPMBillingProjectDeleteRunner.scala

```diff
@@ -92,16 +92,14 @@ class BPMBillingProjectDeleteRunner(
   ): Future[JobStatus] =
     Try(workspaceManagerDAO.getDeleteLandingZoneResult(jobId.toString, lzId, ctx)) match {
       case Failure(e: ApiException) =>
-        val (msg, jobStatus) = e.getCode match {
-          case 500 => (s"Landing Zone deletion operation with jobId $jobId failed: ${e.getMessage}", Incomplete)
-          case 404 => (s"Unable to find jobId $jobId in WSM for Landing Zone deletion", Complete)
+        e.getCode match {
+          case 404 =>
+            val msg = s"Unable to find jobId $jobId in WSM for Landing Zone deletion"
+            billingRepository.updateCreationStatus(projectName, DeletionFailed, Some(msg)).map(_ => Complete)
           case code =>
```

Comment:
I'm not sure retrying a 403 will change anything. 400s and 401s also may not be worth retrying